sites/test-env/package.json

```
@@ -6,7 +6,6 @@
 		"build": "evidence build",
 		"build:strict": "evidence build:strict",
```

update the build command to get it to go on netlify 
```suggestion
		"build:strict": "cross-env NODE_OPTIONS=--max-old-space-size=8192 vite build",

```

---

I actually inlined that into netlify, and the approach worked also - this seems cleaner though

why use vite build not evidence build? Does it matter

---

sorry, I just copied and pasted from the example project, but it should be evidence build!